DataConnectors/Ubiquiti/Connector_Ubiquiti_agent.json

```diff
@@ -0,0 +1,165 @@
+{
+    "id": "Ubiquiti",
+    "title": "Ubiquiti",
+    "publisher": "Ubiquiti",
+    "descriptionMarkdown": "The [Ubiquiti](https://www.ui.com/) data connector provides the capability to ingest Ubiquiti events into Azure Sentinel. Refer to [Ubiquiti  documentation](https://help.ui.com/hc/en-us/articles/204959834-UniFi-How-to-View-Log-Files) for more information.",

```

Comment:
Would be great to add specific logs that this data connector will ingest as well in addition to referring to this as Ubiquity Unifi.

Reply:
updated